usegalaxy.org/find_subsequences.yml.lock

```diff
@@ -0,0 +1,10 @@
+install_repository_dependencies: true
```

Comment:
From [a look in the TS](https://toolshed.g2.bx.psu.edu/view/bgruening/find_subsequences/d882a0a75759) this tool is pretty old and has a `tool_dependencies.xml`. Disabling `install_tool_dependencies` prevents the TS dependencies from being built/installed but `install_repository_dependencies` still causes the repo's `package_*` dependent repos to be installed, which we don't want - there are biopython 1.65 packages in bioconda. In this instance only, can you set `install_repository_dependencies: false`? Normally we would want this option enabled to install suites (although I haven't tested that...).